src/dragonball/src/dbs_virtio_devices/src/vhost/vhost_user/connection.rs

```diff
@@ -189,6 +192,9 @@ impl Endpoint {
         Ok(Some(features))
     }
 
+    /// TODO: Mark this function as unused as it is called by
+    /// vhost-user-fd only.
```

Comment:
what is vhost-user-fd? typo?

Reply:
Yes, patch for vhost-user-fs was merged. These lines are able to be removed.